### PR TITLE
Add support for hosting an imodel view in a modeless dialog

### PR DESCRIPTION
**Pull request by bsteinbk**
- Added ContentDialogManager, ContentDialog, and FloatingViewportContent to allow imodel viewport in a modeless dialog.
 - Fixed styling of Select in Maplayers widget
 - Added example tool to open a floating viewport
 - Added ability to keep from allowing dialog to be dragged where titlebar was not visible.
 - Reset Auto-hide time when mouse is moving in floating content view.
 
See https://github.com/iTwin/itwinjs-core/discussions/2231

### CODE REVIEW

**Review by pmconne — CHANGES_REQUESTED** [review]
This sounds like a very useful feature (wasn't it requested by a user?) Please include some info + doc links in NextVersion.md.

File: ui/appui-react/src/appui-react/configurableui/ConfigurableUiManager.ts [hunk 1]
```diff
@@ -85,8 +86,9 @@ export class ConfigurableUiManager {
     // Initialize the ToolSettingsManager that manages Tool Settings properties.
     ToolSettingsManager.initialize();
 
-    // Initialize the modeless dialog manager.
+    // Initialize dialog managers that require a
```
Comment (NancyMcCallB) [inline ui/appui-react/src/appui-react/configurableui/ConfigurableUiManager.ts:89]: That require a what?

Comment (bsteinbk) [reply ui/appui-react/src/appui-react/configurableui/ConfigurableUiManager.ts]: updated

**Review by NancyMcCallB — APPROVED** [review]

**Comment by bsteinbk** [comment]
@pmconne - added learning doc and nextversion entry.

File: docs/learning/ui/appui-react/ContentViews.md [hunk 2]
```diff
@@ -209,6 +217,21 @@ const fourQuadrants: ContentLayoutProps = {
 };
 ```
 
+### Floating content view
+
+In addition to the "fixed" content views that are defined for a Frontstage, via the [ContentGroupProps]($appui-react), one or more "floating" content dialogs can be opened to display an IModel view, as seen below.
+
+![Floating iModel Content Dialog](./images/ModalDialog.png "Floating iModel Content Dialog")
```
Comment (pmconne) [inline docs/learning/ui/appui-react/ContentViews.md:224]: Path to image is incorrect. Please also include image in NextVersion.md.

**Review by mdastous-bentley — APPROVED** [review]

**Pull request merged**